src/base/bittorrent/trackerentry.cpp

```diff
@@ -77,7 +75,7 @@ int TrackerEntry::tier() const
 
 TrackerEntry::Status TrackerEntry::status() const
 {
-    if (isWorking())
+    if (isWorking() && nativeEntry().verified)
```

Comment:
Is this necessary? What if you don't add it?

Reply:
If not added then not contacted tracker will say working. We don't want that.